commands/incr.md

```diff
@@ -69,25 +69,32 @@ The more simple and direct implementation of this pattern is the following:
 FUNCTION LIMIT_API_CALL(ip)
 ts = CURRENT_UNIX_TIME()
 keyname = ip+":"+ts
+WATCH(keyname)
 current = GET(keyname)
 IF current != NULL AND current > 10 THEN
     ERROR "too many requests per second"
```

Comment:
Need to `UNWATCH()`. Every `WATCH` need a matching `UNWATCH`, `EXEC`, or `DISCARD`